api-documentation.js

```diff
@@ -194,12 +230,26 @@ class ApiDocumentation extends AmfHelperMixin(LitElement) {
       .endpoint="${_docsModel}"
       .previous="${prev}"
       .next="${next}"
-      .baseUri="${baseUri}"
+      .baseUri="${this._getBaseUri()}"
       ?graph="${graph}"
       ?noNavigation="${noBottomNavigation}"
       ></api-endpoint-documentation>`;
   }
 
+  _getBaseUri() {
+    const { baseUri, selectedServerValue } = this;
+
+    return baseUri || selectedServerValue;
+  }
+
+  _attachListeners() {
```

Comment:
The `_attachListeners()` function has a single argument which is `node` on which you are attaching the event. Don't attach the listener to this as the event probably will be dispatched by the request panel which is not in light/shadow DOM of this element.
Please, see this section in [events-target-mixin docs](https://github.com/advanced-rest-client/events-target-mixin#in-a-web-component)